.github/workflows/make-release.yml

```
@@ -0,0 +1,48 @@
+name:  Image release
+
+on:
+  push:
+    tags:
+      - '*'
```

```suggestion
      - 'v**'
```
Maybe only release when the tag starts with `v` (to allow other tags as well)?